### PR TITLE
Rewrite of the multirotor section in the design document

### PR DESCRIPTION
**Pull request by FSXAC**
**Progress**:
- [x] Mechanical (CAD section still requires content but will be added as a separate PR) (0.5h)
- [x] Propulsion (1h)
- [x] Battery and electrical (1h)
- [X] ~FCU and sensors (1h) (merge from Ardell's work)~ (see below)
- [x] Comm and ctrl (1.5h)

- Will create a separate PR to merge FCU content into the doc

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,4 @@ _site/*
 # Ignore mac files
 assets/.DS_Store
 .DS_Store
+docs/deliverables/deliverables.pdf
```
Comment (peterdeutsch) [inline .gitignore:32]: Thanks

File: docs/design/3_2_multirotor_v2.tex [hunk 2]
```diff
@@ -1,26 +1,231 @@
-A remote-piloted aerial system (RPAS) or sometimes referred to as ``drone" consists of serveral subsystems: airframe, mechanical structures, propulsion, battery, sensors, and flight computer. This section will talk about these subsystems in more detail.
+% TODO: replace this image with our own renders
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.85\textwidth]{img/testrigcad1.png}
+	\caption[Team 109 Testing Rig Prototype]{Team 109 Testing Rig Prototype using RPAS Generously Provided by Yiyi Yan and UBC \textit{Unmanned Aerial Systems}}
+	\label{fig:testrigcad1}
+\end{figure}
+
+A remote-piloted aerial system (RPAS) or sometimes referred to as ``drone" and ``multirotor" consists of serveral subsystems: airframe, mechanical structures, propulsion, battery, sensors, and flight computer. This section will explore these subsystems in more detail. Figure~\ref{fig:testrigcad1} is a render of the completed mutlirotor with the computing platform payload attached.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:9]:
```suggestion
A remote-piloted aerial system (RPAS) or sometimes referred to as ``drone" and ``multirotor" consists of serveral subsystems: the airframe, mechanical structures, propulsion, battery, sensors, and flight computer. This section will explore these subsystems in more detail. Figure~\ref{fig:testrigcad1} is a render of the completed multirotor with the computing platform payload attached.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 3]
```diff
@@ -1,26 +1,231 @@
...
+
+\subsubsection{Mechanical}\label{section:drone-mech}
+The mechanical subsystem of the multirotor is comprised of the frame that holds all the propulsion system, on-board computer and sensors, and the payload together.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:12]:
```suggestion
The mechanical subsystem of the multirotor is comprised of the frame that holds the propulsion system, on-board computer/sensors, and the payload together.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 4]
```diff
@@ -1,26 +1,231 @@
...
+		\includegraphics[scale=0.4]{img/drone_x_yaw}
+		\caption{Yaw left}
+		\label{fig:x-yaw}
+	\end{subfigure}
+	
+	\caption{Using differential thrust to obtain 6-DOF movement. }
+	\label{fig:rpas_6dof}
+\end{figure}
+
+A multirotor has several common configurations such as tricopter, quadcopter, hexacopter as depicted in Figure~\ref{fig:rpas-config}. Because each frame has different layouts of motors, the forces acting on the drone is inherently different and thus provides advantages and disadvantages of their own.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:77]:
```suggestion
A multirotor has several common configurations such as tricopter, quadcopter, hexacopter as depicted in Figure~\ref{fig:rpas-config}. As each frame has different motor layouts, the forces acting on the multirotor are inherently different and thus each configuration provides particular advantages and disadvantages.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 5]
```diff
@@ -1,26 +1,231 @@
...
+
+We use a quadcopter X4 configuration (4 motors and propellers in the shape of a letter X) for our multirotor as it is easy to control and provides abundant control system robustness. Such configuration is also the most popular amongst the UAS industry because it is balances redundancy and robustness with capacity and performance.
+
+As seen in Figure \ref{fig:quadcopter-x-t}. Two of the motors spin clockwise and the other two spin counter-clockwise --- effectively canceling each other's unintended torque applied to the multirotor. Applying the same power into each motor allows the MRPAS to hover in place. We can perform 6 degree-of-freedom (DOF) movements by applying a combination of differential thrusts to each motor (Figure \ref{fig:rpas_6dof}). An X4 configuration is inherently stable.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:81]:
```suggestion
An example of the X4 configuration is seen in Figure \ref{fig:quadcopter-x-t}. Two of the motors spin clockwise and the other two spin counter-clockwise --- effectively canceling each other's unintended torque to the multirotor. Applying the same power into each motor allows the MRPAS to hover in place. We can perform 6 degree-of-freedom (DOF) movements by applying a combination of differential thrusts to each motor (Figure \ref{fig:rpas_6dof}). An X4 configuration is thus inherently stable.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 6]
```diff
@@ -1,26 +1,231 @@
...
+
+A dual-blade helicopter, for example, requires fewest number of motors. But like a single-blade helicopter, the system in control theory sense, is naturally unstable and requires difficulty to stabilize the system. A tricopter has odd number of motors which also shares similar characteristics with a helicopter where the system requires an additional tail motor to compensate for the uneven torque.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:83]:
```suggestion
A dual-blade helicopter, for example, requires the fewest number of motors. Like a single-blade helicopter, the system (in control theory sense) is naturally unstable and instills difficulty to stabilize the system. A tricopter has an odd number of motors --- sharing similar characteristics with a helicopter where the system requires an additional tail motor to compensate for the uneven torque. 
```

File: docs/design/3_2_multirotor_v2.tex [hunk 7]
```diff
@@ -1,26 +1,231 @@
...
+
+A dual-blade helicopter, for example, requires fewest number of motors. But like a single-blade helicopter, the system in control theory sense, is naturally unstable and requires difficulty to stabilize the system. A tricopter has odd number of motors which also shares similar characteristics with a helicopter where the system requires an additional tail motor to compensate for the uneven torque. 
+
+Configurations using more than 4 motors such as a hexacopter or an octocopter are very expensive for the amount of parts they require. Drones of these configuration classes are very stable with failure redundancy. However, these drones are typically much larger (0.6m to 1.5m wide) and much heavier (up to 25 kg). As a result, they are typically used in industrial, filming, and agriculture applications with a typical unit cost of at least several thousand dollars. These configurations are not suitable for our applications.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:85]:
```suggestion
Configurations using more than 4 motors (such as hexacopters or octocopters) are very expensive for the amount of parts they require, however they are very stable and contain multiple failure redundancies. These drones are typically much larger (0.6m to 1.5m wide) and much heavier (up to 25 kg). As a result, they are typically used in industrial, filming, and agricultural applications with a typical unit cost of several thousand dollars. These configurations are not suitable for our applications.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 8]
```diff
@@ -1,26 +1,231 @@
...
+		\caption{DJI Spreadwings S1000 (1000 mm)}
+		\label{fig:djis1000}
+	\end{subfigure}
+	
+	\caption{Air Frame Options with Various Size Tiers }
+\end{figure}
+
+Multirotor airframe sizes are commonly designated by their \textit{motor-to-motor-diagonal-span} (MMDS), which measures the distance between the pair of motors which are the furthest apart. 
+
+We adopt a X4 560 mm MMDS carbon fibre frame from JMT, as shown in Figure~\ref{fig:jmtx4}. The frame is carbon fibre to enhance structural rigidity while keeping the system relatively lightweight. The frame features foldable arms and landing gears for portability, but we consider replacing the hinges with static parts to reduce  weight and achieve a longer flight time. The X4 560 mm frame is also wide enough to support propellers up to 14 inches in diameter. The carbon fibre material is light, strong, and rigid. Although this means that the items mounted on the frame would suffer from lack of dampening of vibrations, we implement compensating dampers for sensors separately.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:123]: "we consider replacing the hinges"

Did we replace hinges in the end?

File: docs/design/3_2_multirotor_v2.tex [hunk 9]
```diff
@@ -1,26 +1,231 @@
...
 \subsubsection{Propulsion}
+
+The propulsion system covers the majority of the multirotor parts list and determines the flight performance of the multirotor. We refer to this subsystem as the ``Drive".
+
 \paragraph{Motor Type}
+
+\underline{\textit{Brushed or brushless?}}
+Multirotors are electrically powered by a battery --- a DC source. Two common types of DC motors are  brushed motors and brushless DC (BLDC) motors. Brushed motors commutate mechanically using brushes whereas brushless motors commutate electrically using electronic controllers that switches phases in the stator coils.
+
+We use  BLDC motors for their extended usable lifespan and efficiency. Brushed motors are not appropriate for our applications for its excessive noise, friction, spark, and heat.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:139]:
```suggestion
We use  BLDC motors for their extended usable lifespan and efficiency. Brushed motors are not appropriate for our applications due to their excessive noise, friction, spark, and heat.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 10]
```diff
@@ -1,26 +1,231 @@
...
+
+\underline{\textit{Inrunner or outrunners?}}
+
+BLDC motors further consists of two variants: \textit{inrunner} and \textit{outrunner}.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:143]:
```suggestion
BLDC motors further consist of two variants: \textit{inrunner} and \textit{outrunner}. 
```

File: docs/design/3_2_multirotor_v2.tex [hunk 11]
```diff
@@ -1,26 +1,231 @@
...
+
+We use  BLDC motors for their extended usable lifespan and efficiency. Brushed motors are not appropriate for our applications for its excessive noise, friction, spark, and heat.
+
+\underline{\textit{Inrunner or outrunners?}}
+
+BLDC motors further consists of two variants: \textit{inrunner} and \textit{outrunner}. 
+
+We choose outrunner BLDC motors for its higher torque output at lower speed  (i.e. outrunner motors have low KV constants; see Appendix~\ref{appendix:droneengine} for KV relationships). The lower speed of the rotor is beneficial for two reasons. One, a motor running at a lower speed is more efficient and produces less high frequency vibrations. Two, the attached propeller can be much larger and cut through the air at a lower speed, since drag forces acting on the blades are proportional to the velocity squared. Outrunner motors do require spin-up time as the rotor is more massive and has more inertia, but this compromise is acceptable as the main operating mode of the multirotor is hovering, which requires little agility.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:145]:
```suggestion
We choose outrunner BLDC motors for their higher torque output at lower speed  (i.e. outrunner motors have low KV constants; see Appendix~\ref{appendix:droneengine} for KV relationships). The lower speed of the rotor is beneficial for two reasons. One, a motor running at a lower speed is more efficient and produces fewer high frequency vibrations. Two, the attached propeller can be much larger and cut through the air at a lower speed, since drag forces acting on the blades are proportional to the velocity squared. While outrunner motors do require additional spin-up time as the rotor is more massive and has more inertia, this compromise is acceptable as the main operating mode of the multirotor is hovering, which requires little agility. 
```

File: docs/design/3_2_multirotor_v2.tex [hunk 12]
```diff
@@ -1,26 +1,231 @@
...
+
+\underline{\textit{Inrunner or outrunners?}}
+
+BLDC motors further consists of two variants: \textit{inrunner} and \textit{outrunner}. 
+
+We choose outrunner BLDC motors for its higher torque output at lower speed  (i.e. outrunner motors have low KV constants; see Appendix~\ref{appendix:droneengine} for KV relationships). The lower speed of the rotor is beneficial for two reasons. One, a motor running at a lower speed is more efficient and produces less high frequency vibrations. Two, the attached propeller can be much larger and cut through the air at a lower speed, since drag forces acting on the blades are proportional to the velocity squared. Outrunner motors do require spin-up time as the rotor is more massive and has more inertia, but this compromise is acceptable as the main operating mode of the multirotor is hovering, which requires little agility. 
+
+The alternatives are inrunner motors. Inrunner motors have the rotor on the inside and outrunner motors have the rotor on the outside. Inrunner rotors are lighter and spin faster with a lower torque which means additional gearboxes are required to extract useful torque\cite{invsoutrunner}.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:147]:
```suggestion
The alternatives are inrunner motors, where the rotor is placed on the inside. Inrunner rotors are lighter and spin faster with a lower torque, thus additional gearboxes are required to extract useful torque\cite{invsoutrunner}.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 13]
```diff
@@ -1,26 +1,231 @@
...
+
+\paragraph{Motor Size and Speed}\label{section:motor-speed}
+
+Commercial outrunner BLDC motors generally have the following design parameters: stator size, KV/KT constants, voltage, maximum rated power, and weight. Please refer to Appendix~\ref{appendix:droneengine} for the relationship between the aforementioned design parameters. 
+
+We choose 700 KV motors with stator size 3508.  According to drone hobbist online sources (https://quadquestions.com/blog/2017/02/22/choose-right-size-motors-drone/, https://oscarliang.com/quadcopter-motor-propeller/), the ideal KV motors to maximize performance for frame sizes larger than 450 mm should be 1200 KV or less. The argument of choosing a lower KV motor follows the point made above to balance torque output and energy loss due to air friction.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:153]: Ensure these are converted to proper citations

File: docs/design/3_2_multirotor_v2.tex [hunk 14]
```diff
@@ -1,26 +1,231 @@
...
+
+Our decision process consists of screening and ranking the best motors through applying priorities as follows:
+
+\begin{enumerate}
+    \item Voltage: We chose a motor that operates between 12.0 V and 20.0 V, which is enough for heavy operations.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:158]:
```suggestion
    \item Voltage: We chose a motor that operates between 12.0 V and 20.0 V, which is sufficient for heavy operations.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 15]
```diff
@@ -1,26 +1,231 @@
...
+
 \paragraph{Propellers}
 
+Common RPAS propellers are constructed using either plastic or carbon fibre. For the final deliverable we utilize carbon fiber propellers to maximize efficiency, flight time, and longevity for our client. 
+
+According to Oscar Liang (TODO: https://www.dronethusiast.com/carbon-fiber-props-vs-plastic/),  carbon fibre propellers are tough and rigid and produces efficient and low-noise flight performance.  While plastic propellers are cheap, abundant, and relatively flexible, the manufacture of plastic propellers is not as precise, which leads to aerodynamic inefficiencies. Plastic propellers often require balancing, where one applies tape to the blades such that the weight on each blade is balanced.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:173]: I don't think we need to cite the name of the guy here

File: docs/design/3_2_multirotor_v2.tex [hunk 16]
```diff
@@ -1,26 +1,231 @@
...
+
 \paragraph{Propellers}
 
+Common RPAS propellers are constructed using either plastic or carbon fibre. For the final deliverable we utilize carbon fiber propellers to maximize efficiency, flight time, and longevity for our client. 
+
+According to Oscar Liang (TODO: https://www.dronethusiast.com/carbon-fiber-props-vs-plastic/),  carbon fibre propellers are tough and rigid and produces efficient and low-noise flight performance.  While plastic propellers are cheap, abundant, and relatively flexible, the manufacture of plastic propellers is not as precise, which leads to aerodynamic inefficiencies. Plastic propellers often require balancing, where one applies tape to the blades such that the weight on each blade is balanced. 
+
+We choose propellers with size 1250. Where 12 denotes 12 inches in diameter, and 50 denotes 5.0 inches in propeller pitch. To obtain maximum thrust and minimize speed (low speed is more desirable, as mentioned in Section \ref{section:motor-speed}), we strike a balance if we choose 11 to 13 inch propellers to be fitted onto the multirotor.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:175]:
```suggestion
We choose propellers with size 1250, where 12 denotes 12 inches in diameter, and 50 denotes 5.0 inches in propeller pitch. To obtain maximum thrust and minimize speed (seeing how low speed is more desirable, as mentioned in Section \ref{section:motor-speed}), we strike a balance if we choose 11 to 13 inch propellers to be fitted onto the multirotor.
```

File: docs/design/3_2_multirotor_v2.tex (hunk 8 above)
Comment (FSXAC) [reply docs/design/3_2_multirotor_v2.tex:123]: Nope, I will rephrase this to suggest possible modification

File: docs/design/3_2_multirotor_v2.tex (hunk 13 above)
Comment (FSXAC) [reply docs/design/3_2_multirotor_v2.tex:153]: yeah, they're "TODO's"

File: docs/design/3_2_multirotor_v2.tex (hunk 15 above)
Comment (FSXAC) [reply docs/design/3_2_multirotor_v2.tex:173]: ok

File: docs/design/3_2_multirotor_v2.tex [hunk 17]
```diff
@@ -1,26 +1,250 @@
-A remote-piloted aerial system (RPAS) or sometimes referred to as ``drone" consists of serveral subsystems: airframe, mechanical structures, propulsion, battery, sensors, and flight computer. This section will talk about these subsystems in more detail.
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.85\textwidth]{img/multirotor.png}
+	\caption[Team 109 Multirotor Drone]{Team 109 Multirotor Drone}
+	\label{fig:drone1}
+\end{figure}
+
+A remote-piloted aerial system (RPAS) or sometimes referred to as ``drone" and ``multirotor" consists of serveral subsystems: the airframe, mechanical structures, propulsion, battery, sensors, and flight computer. This section will explore these subsystems in more detail. Figure~\ref{fig:drone1} is a render of the completed multirotor with the computing platform payload attached.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:8]:
```suggestion
A remote-piloted aerial system (RPAS), sometimes referred to as a ``drone" or ``multirotor", consists of several subsystems: the airframe, mechanical structures, propulsion, battery, sensors, and flight computer. This section will explore these subsystems in more detail. Figure~\ref{fig:drone1} is a render of the completed multirotor with the computing platform payload attached.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 18]
```diff
@@ -1,26 +1,250 @@
...
+
+\subsubsection{Mechanical}\label{section:drone-mech}
+
+The frame and mounts which holds the propulsion system, on-board computer/sensors, and the payload together makes up most of the mechanical subsystem of the multirotor.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:12]:
```suggestion
The frame and mounts which hold the propulsion system, on-board computer/sensors, and payload together makes up most of the mechanical subsystem of the multirotor.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 19]
```diff
@@ -1,26 +1,250 @@
...
+		\includegraphics[scale=0.4]{img/drone_x_yaw}
+		\caption{Yaw left}
+		\label{fig:x-yaw}
+	\end{subfigure}
+	
+	\caption{Using differential thrust to achive 6-DOF movement. }
+	\label{fig:rpas_6dof}
+\end{figure}
+
+Multirotors have different configurations such as tricopter, quadcopter, hexacopter as depicted in Figure~\ref{fig:rpas-config}. As each frame has different motor layouts, the forces acting on the multirotor are inherently different and thus each configuration provides particular advantages and disadvantages.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:77]:
```suggestion
Multirotors have different configurations, such as tricopter, quadcopter, hexacopter, as depicted in Figure~\ref{fig:rpas-config}. As each frame has different motor layouts, the forces acting on the multirotor are inherently different and thus each configuration provides particular advantages and disadvantages.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 20]
```diff
@@ -1,26 +1,250 @@
...
+
+We use a quadcopter X4 configuration (4 motors and propellers in the shape of a letter X) for our multirotor because it is easy to control and provides abundant control system robustness. Such configuration is also the most popular amongst the UAS industry because for its balance between redundancy and robustness with capacity and performance.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:79]:
```suggestion
We use a quadcopter X4 configuration (4 motors and propellers in the shape of a letter X) for our multirotor because it is easy to control and provides abundant control system robustness. Such configuration is also the most popular amongst the UAS industry for its balance between redundancy and robustness with capacity and performance.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 21]
```diff
@@ -1,26 +1,250 @@
...
+
+Figure \ref{fig:quadcopter-x-t} shows an example of the X4 configuration. Two of the motors spin clockwise and the other two spin counter-clockwise --- effectively canceling each other's unintended torque to the multirotor. Applying the same power into each motor allows the MRPAS to hover in place. We can perform 6 degree-of-freedom (DOF) movements by applying a combination of differential thrusts to each motor (Figure \ref{fig:rpas_6dof}). An X4 configuration is thus inherently stable.
+
+A dual-blade helicopter, for example, requires the fewest number of motors. Like a single-blade helicopter, the system (in control theory sense) is naturally unstable and instills difficulty to stabilize the system. A tricopter has an odd number of motors --- sharing similar characteristics with a helicopter where the system requires an additional tail motor to compensate for the uneven torque.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:83]:
```suggestion
A dual-blade helicopter, for example, requires the fewest number of motors. Like a single-blade helicopter, the system (in a control theory sense) is naturally unstable and instills difficulty to stabilize the system. A tricopter has an odd number of motors --- sharing similar characteristics with a helicopter where the system requires an additional tail motor to compensate for the uneven torque. 
```

File: docs/design/3_2_multirotor_v2.tex [hunk 22]
```diff
@@ -1,26 +1,250 @@
...
+
+A dual-blade helicopter, for example, requires the fewest number of motors. Like a single-blade helicopter, the system (in control theory sense) is naturally unstable and instills difficulty to stabilize the system. A tricopter has an odd number of motors --- sharing similar characteristics with a helicopter where the system requires an additional tail motor to compensate for the uneven torque. 
+
+Configurations using more than 4 motors (such as hexacopters or octocopters) are very expensive for the amount of parts they require, however they fly more stablily and has failure redundancies. These drones are typically much larger (0.6m to 1.5m wide) and much heavier (up to 25 kg). As a result, they are typically used in industrial, filming, and agricultural applications with a typical unit cost of thousands of dollars. These configurations are not suitable for our applications.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:85]:
```suggestion
Configurations using more than 4 motors (such as hexacopters or octocopters) are very expensive for the amount of parts they require, however their flight stability is higher and they have multiple failure redundancies. These drones are typically much larger (0.6m to 1.5m wide) and much heavier (up to 25 kg). As a result, they are typically used in industrial, filming, and agricultural applications with a typical unit cost of thousands of dollars. These configurations are not suitable for our applications.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 23]
```diff
@@ -1,26 +1,250 @@
...
+\paragraph{Customized Add-ons}
+% TODO: add CAD parts
+
 \subsubsection{Propulsion}
+
+The propulsion system covers the majority of the multirotor parts list and determines the flight performance of the multirotor. We refer to this subsystem as the ``Drive".
+
 \paragraph{Motor Type}
+
+Multirotors are electrically powered by a battery --- a DC source. Two common types of DC motors are  brushed motors and brushless DC (BLDC) motors. Brushed motors commutate mechanically using brushes whereas brushless motors commutate electrically using electronic controllers that switches phases in the stator coils. We use BLDC motors for their extended usable lifespan and efficiency. Brushed motors are not appropriate for our applications due to their excessive noise, friction, spark, and heat.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:136]:
```suggestion
Multirotors are electrically powered by a battery --- a DC source. Two common types of DC motors are  brushed motors and brushless DC (BLDC) motors. Brushed motors commutate mechanically using brushes whereas brushless motors commutate electrically using electronic controllers that switches phases in the stator coils. We use BLDC motors for their extended usable lifespan and efficiency. Brushed motors are not appropriate for our applications due to their excessive noise, friction, and heat.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 24]
```diff
@@ -1,26 +1,250 @@
...
+
+BLDC motors further consist of two variants: \textit{inrunner} and \textit{outrunner}. We choose outrunner BLDC motors for their higher torque output at lower speed  (i.e. outrunner motors have low KV constants; see Appendix~\ref{appendix:droneengine} for KV relationships). The lower speed of the rotor is beneficial for two reasons. One, a motor running at a lower speed is more efficient and produces fewer high frequency vibrations. Two, the attached propeller can be much larger and cut through the air at a lower speed, since drag forces acting on the blades are proportional to the velocity squared. While outrunner motors require more spin-up time as the rotor is more massive and has more inertia, this compromise is acceptable as the main operating mode of the multirotor is hovering, which requires little agility. 
+
+The alternatives are inrunner motors, where the rotor is on the inside. Inrunner rotors are lighter and spin faster with a lower torque, thus requires gearboxes to extract useful torque\cite{invsoutrunner}.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:140]:
```suggestion
The alternative is inrunner motors, where the rotor is on the inside. Inrunner rotors are lighter and spin faster with a lower torque, thus requires gearboxes to extract useful torque\cite{invsoutrunner}.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 25]
```diff
@@ -1,26 +1,250 @@
...
+
+\paragraph{Motor Size and Speed}\label{section:motor-speed}
+
+Commercial outrunner BLDC motors generally have the following design parameters: stator size, KV/KT constants, voltage, maximum rated power, and weight. Please refer to Appendix~\ref{appendix:droneengine} for the relationship between the aforementioned design parameters. 
+
+We choose 700 KV motors with stator size 3508.  According to drone hobbist online sources \cite{kv1, kv2}, the ideal KV motors to maximize performance for frame sizes larger than 450 mm should be 1200 KV or less. The argument of choosing a lower KV motor follows the point made above to balance torque output and energy loss due to air friction.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:146]:
```suggestion
We chose 700 KV motors with stator size 3508.  According to drone hobbist online sources \cite{kv1, kv2}, the ideal KV motors to maximize performance for frame sizes larger than 450 mm should be 1200 KV or less. The argument of choosing a lower KV motor follows the point made above to balance torque output and energy loss due to air friction.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 26]
```diff
@@ -1,26 +1,250 @@
...
+
+Our decision process consists of screening and ranking the best motors through applying priorities as follows:
+
+\begin{enumerate}
+    \item Voltage: We chose a motor that operates between 12.0 V and 20.0 V, which is sufficient for heavy operations.
+    \item KV constant: Since the motors' RPM-per-volt ratio (KV constant) is inversely proportional of torque-per-current ratio (KT constant). We chose motors that have KV constants between 500 KV and 1000 KV. These constants are considered low-speed and high-torque, and are typically used for large drones and endurance flights. We also chose this range such that the propellers can spin slower and more efficiently, extending the maximum flight time.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:152]:
```suggestion
    \item KV constant: Since the motors' RPM-per-volt ratio (KV constant) is inversely proportional of torque-per-current ratio (KT constant), we chose motors that have KV constants between 500 KV and 1000 KV. These constants are considered low-speed and high-torque, and are typically used for large drones and endurance flights. We also chose this range such that the propellers can spin slower and more efficiently, extending the maximum flight time.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 27]
```diff
@@ -1,26 +1,250 @@
...
+
+$$
+I_{max}=30 \times 5000 = 150~\text{A}
+$$
+
 \paragraph{Power Distribution}
 
+Figure \ref{fig:pdb} depicts the layout of electrical power distribution for the multirotor subsystems. The 11.0 to 12.0 V from the 3S battery (or 14.0 to 15.0V of 4S batteries) is connected to a central PCB for power distribution. All the ESCs are to be connected in parallel to draw current from the battery. The power distribution board also has built-in voltage regulators for 5.0 V supply for digital flight hardware and other peripherals. The power distribution board is sandwiched between the flight controller unit/autopliot and the air frame as shown in Figure \ref{fig:pdb2}. Do not confuse the power distribution with the power supply for the payload. The power distribution shown here only supplies power to the multirotor subsystems and is independent of the PLB and PMB operations.
```
Comment (peterdeutsch) [inline docs/design/3_2_multirotor_v2.tex:218]:
```suggestion
Figure \ref{fig:pdb} depicts the layout of electrical power distribution for the multirotor subsystems. The 11.0 to 12.0 V from the 3S battery (or 14.0 to 15.0V of 4S batteries) is connected to a central PCB for power distribution. All the ESCs are to be connected in parallel to draw current from the battery. The power distribution board also has built-in voltage regulators for 5.0 V supply to digital flight hardware and other peripherals. The power distribution board is sandwiched between the flight controller unit/autopliot and the air frame as shown in Figure \ref{fig:pdb2}. Do not confuse the power distribution with the power supply for the payload. The power distribution shown here only supplies power to the multirotor subsystems and is independent of the PLB and PMB operations.
```

File: docs/design/3_2_multirotor_v2.tex [hunk 28]
```diff
@@ -1,26 +1,250 @@
...
+		\centering
+		\includegraphics[width=\textwidth]{img/djis1000}
+		\caption{DJI Spreadwings S1000 (1000 mm)}
+		\label{fig:djis1000}
+	\end{subfigure}
+	
+	\caption{Air Frame Options with and Their Sizes }
+\end{figure}
+
+Multirotor airframe sizes are commonly designated by their \textit{motor-to-motor-diagonal-span} (MMDS), which measures the distance between the pair of motors which are the furthest apart.
```
Comment (wzfftxwd) [inline docs/design/3_2_multirotor_v2.tex:121]:
```suggestion
Multirotor airframe sizes are commonly designated by their \textit{motor-to-motor-diagonal-span} (MMDS), which is the distance between the pair of motors which are the furthest apart. 
```

File: docs/design/3_2_multirotor_v2.tex [hunk 29]
```diff
@@ -1,26 +1,250 @@
...
+		\caption{DJI Spreadwings S1000 (1000 mm)}
+		\label{fig:djis1000}
+	\end{subfigure}
+	
+	\caption{Air Frame Options with and Their Sizes }
+\end{figure}
+
+Multirotor airframe sizes are commonly designated by their \textit{motor-to-motor-diagonal-span} (MMDS), which measures the distance between the pair of motors which are the furthest apart. 
+
+We adopt a X4 560 mm MMDS carbon fibre frame from JMT, as shown in Figure~\ref{fig:jmtx4}. The frame is carbon fibre to enhance structural rigidity while keeping the system lightweight. The frame features foldable arms and landing gears for portability. Our client has the option to replace the brass hinges with static parts to reduce  weight and achieve a longer flight time. The X4 560 mm frame is also wide enough to support propellers up to 14 inches in diameter. The carbon fibre material is light, strong, and rigid. Although this means that the items mounted on the frame would suffer from lack of dampening of vibrations, we add compensating dampers for sensors separately.
```
Comment (wzfftxwd) [inline docs/design/3_2_multirotor_v2.tex:123]:
```suggestion
We adopted an X4 560 mm MMDS carbon fibre frame from JMT, as shown in Figure~\ref{fig:jmtx4}. The frame is made of carbon fibre to enhance structural rigidity while keeping the system lightweight. The frame features foldable arms and landing gears for portability. Our client has the option to replace the brass hinges with static parts to reduce weight and achieve a longer flight time. The X4 560 mm frame is also wide enough to support propellers up to 14 inches in diameter. The carbon fibre material is light, strong, and rigid. This means that the items mounted on the frame would suffer from the lack of dampening of vibrations. Therefore, we added compensating dampers for sensors separately.
```
